include/lo2s/monitor/metric_monitor.hpp

```diff
@@ -21,35 +21,28 @@
 
 #pragma once
 
-#include <lo2s/perf/tracepoint/format.hpp>
 #include <lo2s/perf/tracepoint/writer.hpp>
 
-#include <lo2s/monitor/fd_monitor.hpp>
-#include <lo2s/pipe.hpp>
-#include <lo2s/time/time.hpp>
+#include <lo2s/monitor/poll_monitor.hpp>
 #include <lo2s/trace/trace.hpp>
 
-#include <thread>
-#include <vector>
+#include <map>
+#include <memory>
 
 namespace lo2s
 {
-namespace perf
+namespace monitor
 {
-namespace tracepoint
-{
-/*
- * NOTE: Encapsulates counters for ALL cpus, totally different than counters!
- *
- * TODO Split up in one Monitor per core and pin it!
- */
-class MetricMonitor : public monitor::FdMonitor
+
+// NOTE: Encapsulates counters for ALL cpus, totally different than counters!
```

Comment:
This note is confusing as the constructor takes a `cpuid` parameter.

@tilsche

Reply:
Seems the note is no longer true.

Also the naming is ambiguous once moved out of the `tracepoint` namespace.

But then again, we eventually just want a `CpuMonitor` which also contains a `tracepoint::Writer`.